functions/_fzf_search_git_status.fish

```diff
@@ -25,7 +25,7 @@ function _fzf_search_git_status --description "Search the output of git status.
                 end
             end
 
-            commandline --current-token --replace -- (string escape -- $cleaned_paths | string join ' ')
```

Comment:
Turns out this is unnecessary because git status already escapes filenames with spaces in them